job_templates/configure_cloud_connector_-_ansible_default.erb

```diff
@@ -7,11 +7,13 @@ template_inputs:
   input_type: user
   advanced: false
   value_type: plain
+  hidden_value: false
```

Comment:
this is a default, so it should not be necessary, right?

Reply:
it is not, however exporting the template would add it (same as e.g. advanced: false above)